docs/src/pages.ts

```diff
@@ -280,15 +280,6 @@ const pages: readonly MuiPage[] = [
       { pathname: '/guides/flow' },
     ],
   },
```

Comment:
I was tempted to remove all the rest, but I didn't to keep this PR relatively small.